examples/README.md

```diff
@@ -2,15 +2,17 @@
 
 ## Note that this example uses Tekton Pipeline resources, so make sure you've [installed](https://github.com/tektoncd/pipeline/blob/master/docs/install.md) that first!
 
-In this example you will use Triggers to create a PipelineRun and PipelineResource that simply clones a GitHub repository and prints a couple of messages.
+In this example you will use Triggers to create a PipelineRun and
+PipelineResource that simply clones a GitHub repository and prints a couple of
+messages.
 
 1. Create the resources for the example
 
 ```yaml
-kubectl apply -f role-resources
-kubectl apply -f triggertemplates/triggertemplate.yaml
-kubectl apply -f triggerbindings/triggerbinding.yaml
-kubectl apply -f eventlisteners/eventlistener.yaml
+kubectl apply -f role-resources kubectl apply -f
```

Comment:
I don't think we want this to line wrap, but I think this is because the codeblock is yaml instead of sh. Can you change this and rerun/revert the indent change?

Reply:
This is all automation, if the tool doesn't do the right thing on the input, then generally you have to find a way of tweaking things so that it does what you want.

I can disable this here, as I did for pipelines, until someone has the patience to get things running cleanly?

Reply:
Fairly sure https://github.com/tektoncd/triggers/pull/303 will address this. We can either wait for that to be submitted, include the changes in this PR, or disable here until it's in. Whichever you're more comfortable with! ^_^

Reply:
it'll just reopen in the morning, but we can ignore it until your change lands.